src/utils/viewer/viewer.cpp

```diff
@@ -239,6 +239,10 @@ Viewer::Viewer(const Arguments& arguments)
       .addOption("physics-config", ESP_DEFAULT_PHYS_SCENE_CONFIG_REL_PATH)
       .setHelp("physics-config",
                "Provide a non-default PhysicsManager config file.")
+      .addOption("object-dir", "./data/objects")
+      .setHelp("object-dir",
+               "Provide a directory to search for object config files "
+               "(relative to habitat-sim directory).")
```

Comment:
This is added to provide a work-around for the removal of similar functionality in physics manager attributes config files.

This will be obviated by dataset support and could be removed, once full MM/Dataset handling functionality is provided.